src/styles/blocks/page.scss

```diff
@@ -0,0 +1,13 @@
+@import "../utils/_variables.scss";
```

Comment:
no need to import it here, you have already imported it to main.scss (all similar cases)

Reply:
When i delete imports from files where i use my variables, my program doesn't work

Reply:
I can delete imports from main.scss instead, it works well